wallet/src/de/schildbach/wallet/ui/WalletActivity.java

```diff
@@ -295,7 +295,8 @@ private void showHideJoinDashPayAction() {
         final Coin walletBalance = wallet.getBalance(Wallet.BalanceType.ESTIMATED);
         boolean canAffordIt = walletBalance.isGreaterThan(Constants.DASH_PAY_FEE)
                 || walletBalance.equals(Constants.DASH_PAY_FEE);
-        boolean visible = syncComplete && canAffordIt && config.getShowJoinDashPay();
+        //FIXME to be fixed after review
+        boolean visible = true ;//syncComplete && canAffordIt && config.getShowJoinDashPay();
```

Comment:
This was changed to make testing easier and **needs to be changed back before merging.**

Reply:
We didn't change this, but can change it later.